wasm-corpus/run.sh

```diff
@@ -8,5 +8,5 @@ if [[ "$1" = "rebuild" ]]; then
 else
   # Optionally takes the args `lang` (ex: wat, rust) and `file` (ex: br.wat)
   # both default to all
-  docker run --platform linux/amd64 -v ./src/test/resources:/usr/code --rm chicory/wasm-corpus $1 $2
+  docker run --platform linux/amd64 -v $(pwd)/src/test/resources:/usr/code --rm chicory/wasm-corpus $1 $2
```

Comment:
Instead of `pwd` which depends on "from where" the script is invoked, I started relying on "where the script is placed" using this at the start:

https://github.com/dylibso/chicory/blob/ddb11ea73a99ffb5b290afe5a70145193191e78c/scripts/compile-resources.sh#L5

Reply:
Thanks i'll follow up with some improvements in the next PR!